google/cloud/spanner_v1/database.py

```diff
@@ -138,6 +142,7 @@ def __init__(
         encryption_config=None,
         database_dialect=DatabaseDialect.DATABASE_DIALECT_UNSPECIFIED,
         database_role=None,
+        drop_protection_enabled=False,
```

Comment:
This method is called from [instance.database()](https://github.com/googleapis/python-spanner/blob/main/google/cloud/spanner_v1/instance.py#L426), you need to add changes there as well

Reply:
Done.